web-local/config-local.yaml

```diff
@@ -58,7 +58,7 @@ systemOfRecord:
   stashRoot: s3://emodb-us-east-1/stash/ci
 
   # Optional property - to exclude the tables satisfying the condition from the daily Stash run.
-  stashBlackListTableConditionString: if(intrinsic(\"~table\":or(\"brandanswers:badgeoverride\",\"brandanswers:badgeoverrideimage\")))
+  stashBlackListTableCondition: {.., "type": "review"}
```

Comment:
This is also why the Jenkins test is failing.  The condition needs to be in quotes since it's a String, such as:

```
  stashBlackListTableCondition: "{..,\"type\":\"value\"}"
```